spalp_example/src/EventSubscriber/SpalpExampleAppIdsAlterSubscriber.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Drupal\spalp_example\EventSubscriber;
+
+use Drupal\spalp\Event\SpalpAppIdsAlterEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class SpalpExampleAppIdsAlterSubscriber.
+ *
+ * @package Drupal\spalp\EventSubscriber
+ */
+class SpalpExampleAppIdsAlterSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents() {
+    return [
+      SpalpAppIdsAlterEvent::EVENT_APP_IDS_ALTER => 'doAppIdsListAlter',
+    ];
+  }
+
+  /**
+   * React to app id alter event to add this modules app id.
+   *
+   * @param \Drupal\spalp\Event\SpalpAppIdsAlterEvent $event
+   */
+  public function doAppIdsListAlter(SpalpAppIdsAlterEvent $event) {
+    $event->ids[SPALP_EXAMPLE_APP_ID] = SPALP_EXAMPLE_APP_ID;
```

Comment:
try to get in the habit of using getters and setters.. 

see examples below

**Event Getter / Setter:** 
https://github.com/nodespark/elasticsearch_connector/blob/8.x-5.x/src/Event/BuildSearchParamsEvent.php#L35

https://github.com/nodespark/elasticsearch_connector/blob/8.x-5.x/src/Event/BuildSearchParamsEvent.php#L44

**Subscriber Getter / Setter:** 
https://github.com/dakkusingh/search_api_elasticsearch_attachments/blob/8.x-1.x/src/EventSubscriber/BuildSearchParams.php#L30

https://github.com/dakkusingh/search_api_elasticsearch_attachments/blob/8.x-1.x/src/EventSubscriber/BuildSearchParams.php#L51